wg-structured-logging/annual-report-2023.md

```diff
@@ -12,15 +12,20 @@
    - white papers
    - work not tracked in KEPs
 -->
+- Graduated [contextual logging](https://github.com/kubernetes/enhancements/issues/3077) as Beta.
+  - All of kube-controller-manager and some parts of kube-scheduler converted (in-tree).
+- klog package updates that support using slog as a backend.
+- Enhance the logcheck tool for better detection of contextual logging and use newer APIs in Kubernetes.
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
+- Our working group is looking for more contributors to help us in the migration of contextual logging. If you are interested or experienced in the area, we would very much welcome your participation.
 
 ## Operational
 
 Operational tasks in [wg-governance.md]:
 
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] WG leaders in [sigs.yaml] are accurate and active, and updated if needed
+- [x] [README.md] reviewed for accuracy and updated if needed
+- [x] WG leaders in [sigs.yaml] are accurate and active, and updated if needed
```

Comment:
@mengjiao-liu: I think you can also tick the two boxed below:
- We stopped doing meetings and focused on Slack, so the "if needed" is satisfied (there was no need...).
- I talked about the state of the WG with SIG Instrumentation folks at KubeCon. No other specific updates to other SIGs, though. They see that we are active as PRs come in which need their approval.

Reply:
I think it's fair to include https://kccnceu2024.sched.com/event/1YhhM/leverage-contextual-and-structured-logging-in-k8s-for-enhanced-monitoring-patrick-ohly-intel-gmbh-shivanshu-raj-shrivastava-independent-mengjiao-liu-daocloud under "Updates provided" as a separate line item.

Reply:
> I think it's fair to include https://kccnceu2024.sched.com/event/1YhhM/leverage-contextual-and-structured-logging-in-k8s-for-enhanced-monitoring-patrick-ohly-intel-gmbh-shivanshu-raj-shrivastava-independent-mengjiao-liu-daocloud under "Updates provided" as a separate line item.

Does it matter that this speech took place in 2024? Or is it best to post links to relevant speeches in 2023?

Reply:
You are right, this report is for 2023. We had planned to do something for KubeCon NA 2023, but had to cancel because of travel restrictions.

Let's remember the link for KubeCon EU 2024 for the next report.

I still think we can tick the box, though.

Reply:
KubeCon China 2023 Naman Lakhwani as the speaker talked about structured logging and contextual logging, I thought we could put this link: https://sched.co/1PTJw

Reply:
I have updated it to the last push.